src/http_checker.h

```diff
@@ -4,6 +4,7 @@
 
 #include <cstdlib>
 #include <string>
+#include <cstring>
```

Comment:
I was also hesitant to add this change till we make sure we are using the correct `node` and `node-gyp` versions.